devices.go

```diff
@@ -364,15 +366,33 @@ func (s *DeviceServiceOp) List(projectID string, opts *ListOptions) (devices []D
 	}
 }
 
+func (d *Device) GetHref() string {
```

Comment:
this is not needed if Device embeds `*Href` (https://github.com/packethost/packngo/pull/280/files#diff-ebde46360e5c028cc3bde9d49c49bfb6038d13cd340af1d2d61845323ec284fbR59-R64), as this PR does with Organization https://github.com/packethost/packngo/pull/280/files#diff-d5c3dbfdac6c7539c18385ddd9c9dceff98dfda42c0bc25acaf87135caf6e066R31

Reply:
I'd do the *Href embedding, it seems like a huge save on amount of code. I don't think it will break too many things as the Href field is not used so much AFAIK.